src/exchange.c

```diff
@@ -47,7 +47,12 @@ static struct {
     DM_ELEM_SHA(384),
     DM_ELEM_SHA(224),
     { "SHA1", CKM_SHA_1, CKD_SHA1_KDF, 20 },
-    { NULL, 0, 0, 0, }
+    {
+        NULL,
+        0,
+        0,
+        0,
+    },
```

Comment:
This is still off, clang-format just doesn't get it... guess will have to sdjust manually before merging

Reply:
Remove the last comma after `0`?

Reply:
uh, could be I guess